react-ui/src/serviceWorker.js

```diff
@@ -2,6 +2,8 @@
 // This optional code is used to register a service worker.
 // register() is not called by default.
 
+import { IS_DEV } from './env';
```

Comment:
Nit: Could you put the import after the comments? It's kind of in the middle of the same comment block (not sure why they didn't use the `/** */` format; would have made more sense here)

Reply:
Done